onnxruntime/core/providers/dml/DmlExecutionProvider/src/MLOperatorAuthorImpl.cpp

```diff
@@ -1748,7 +1709,9 @@ namespace Windows::AI::MachineLearning::Adapter
         }
         else
         {
-            m_abiDataInterface.CopyTo(dataInterface);
+            auto bufferRegion = GetBufferRegion();
+            bufferRegion.GetD3D12Resource()->AddRef();
+            *dataInterface = bufferRegion.GetD3D12Resource();
```

Comment:
CopyTo?

Reply:
`GetD3D12Resource()` returns a raw pointer, so there's no `ComPtr` to automatically do the `AddRef`